apps/cookbook/src/app/showcase/page-showcase/page-showcase.component.html

```diff
@@ -51,13 +51,25 @@ <h2>Page with Kirby Router Outlet</h2>
     Using page with the <code>kirby-router-outlet</code> will cause the views to be stacked instead
     of replaced.
   </p>
-
   <h3>Life-cycle hooks</h3>
+
+  <p>
+    The <code>kirby-router-outlet</code> stacks the pages/components when it navigates. This means
+    that when one component is stacked on top of another, the component on the bottom won't be
+    destroyed. This means that eg. subscriptions aren't unsubscribed. This can result in severe
```

Comment:
```suggestion
    that when one component is stacked on top of another, the component on the bottom won't be
    destroyed. This means that the <code>ngOnDestroy</code> hook will not run as expected, and any logic for eg. unsubscribing from subscriptions is not taking place. This can result in severe
```